apps/omg_watcher/test/integration/block_getter_test.exs

```diff
@@ -275,6 +275,83 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
     JSONRPC2.Servers.HTTP.shutdown(BadChildChainTransaction)
   end
 
+  @tag fixtures: [:watcher_sandbox, :stable_alice, :child_chain, :token, :stable_alice_deposits]
+  test "transaction which is using already spent utxo from exit and happened after m_sv causes to emit invalid_block event",
+       %{stable_alice: alice, stable_alice_deposits: {deposit_blknum, _}} do
+    # TODO remove this tx , use directly deposit_blknum to get_exit_data
+    tx = API.TestHelper.create_encoded([{deposit_blknum, 0, 0, alice}], @eth, [{alice, 10}])
+    {:ok, %{blknum: deposit_blknum}} = Client.call(:submit, %{transaction: tx})
+
+    bad_tx = API.TestHelper.create_recovered([{deposit_blknum, 0, 0, alice}], @eth, [{alice, 10}])
+
+    %{hash: bad_block_hash, number: bad_block_number, transactions: _} =
+      bad_block = API.Block.hashed_txs_at([bad_tx], 40_000)
+
+    {:module, BadChildChainBLock, _, _} = OMG.Watcher.Integration.BadChildChainBLock.create_module(bad_block)
+
+    JSONRPC2.Servers.HTTP.http(BadChildChainBLock, port: BadChildChainBLock.port())
+
+    {:ok, _, _socket} = subscribe_and_join(socket(), Channel.Byzantine, "byzantine")
+
+    {:ok, _} =
```

Comment:
explain the trick here.
Also I think a 1-line would look better for this `submit_block` call

Reply:
1) "#   Here we manually submiting invalid block with big/future nonce to the Rootchain to make 
 the Rootchain to mine invalid block instead of block submitted by child chain ,"

2) It is not possible to move to 1-line cause `submit_block` fun is using `bad_block_hash` var which is calucalted by this code:
` %{hash: bad_block_hash, number: bad_block_number, transactions: _} =
      bad_block = API.Block.hashed_txs_at([bad_tx], bad_block_number)` 
I moved `submt_block` call after this code.

Reply:
2. I just meant the formatting `{:ok, _} = OMG.Eth.RootChain.submit_block(bad_block_hash, 40, 1)`, that should be <120 chars :)